<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fight</class>
 <widget class="QWidget" name="Fight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>战斗</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Pokemon_Stage_3_Client.qrc">
    <normaloff>:/Resources/Icon.ico</normaloff>:/Resources/Icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Resources/Fight.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelSystemPokemonImg">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelMyPokemonImg">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>240</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 0.5);</string>
   </property>
   <property name="text">
    <string>生命值
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBarSystemHP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {border:2px solid grey;border-radius:8px; background-color:#FFFFFF;}
QProgressBar::chunk{border-radius:6px;background-color:rgb(255,0,0);}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 0.5);</string>
   </property>
   <property name="text">
    <string>生命值
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBarMyHP">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {border:2px solid grey;border-radius:8px; background-color:#FFFFFF;}
QProgressBar::chunk{border-radius:6px;background-color:rgb(255,0,0);}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelSystemName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 0.5);</string>
   </property>
   <property name="text">
    <string>System</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelMyName">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>410</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 0.5);</string>
   </property>
   <property name="text">
    <string>Me</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSystemHP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>HP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelMyHP">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>HP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSystemAttackMethod">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 0.5);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelMyAttackMethod">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>200</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 0.5);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Pokemon_Stage_3_Client.qrc"/>
 </resources>
 <connections/>
</ui>
